app/src/panels/RequestSalary/AllocationFields.js

```diff
@@ -0,0 +1,222 @@
+
+  return (
+    <div>
+      <div
+        css={`
+          display: flex;
+          align-items: center;
+          justify-content: space-between;
+          margin-bottom: ${1 * GU}px;
+        `}
+      >
+        <AllocationInput
+          text="Base asset percentage"
+          value={convertedBaseAllocation}
+          onChange={handleBaseAllocationChange}
+        />
+        <AllocationInput
+          text="Equity asset percentage"
+          value={convertedEquityAllocation}
+          onChange={handleEquityAllocationChange}
+          css="text-align:right"
+        />
+      </div>
+      <CostumSlider
+        value={convertedBaseAllocation / 100}
+        onUpdate={handleSliderChange}
+        colors={{
+          baseAsset: BASE_ASSET_COLOR,
+          equityAsset: EQUITY_ASSET_COLOR,
+        }}
+      />
+      <div
+        css={`
+          margin-top: ${2 * GU}px;
+          display: flex;
+          justify-content: space-between;
+        `}
+      >
+        <div>
+          <AssetLabel color={BASE_ASSET_COLOR} label={baseAsset.symbol} />
+          <span>
+            {formatTokenAmount(baseAssetTotalAmount, false, baseAsset.decimals)}{' '}
+            {baseAsset.symbol}
+          </span>
+        </div>
+        <div
+          css={`
+            text-align: right;
+          `}
+        >
+          <AssetLabel color={EQUITY_ASSET_COLOR} label="EQUITY" />
+          <span>
+            {formatTokenAmount(
+              equityAssetTotalAmount,
+              false,
+              baseAsset.decimals
+            )}{' '}
+            {equityAsset.symbol}
+          </span>
+        </div>
+      </div>
+    </div>
+  )
+}
+
+const AllocationInput = ({ text, value, onChange, ...props }) => {
+  const theme = useTheme()
+  return (
+    <div {...props}>
+      <span
+        css={`
+          ${textStyle('label2')};
+          color: ${theme.contentSecondary};
+          display: block;
+        `}
+      >
+        {text}
+      </span>
+      <TextInput
+        name="Equity allocation"
+        value={value}
+        onChange={onChange}
+        type="number"
+        css={`
+          width: ${9 * GU}px;
+          margin-right: ${0.5 * GU}px;
+          margin-top: ${1 * GU}px;
+        `}
+      />
+      %
+    </div>
+  )
+}
+
+const AssetLabel = ({ color, label }) => {
+  const theme = useTheme()
+
+  return (
+    <div
+      css={`
+        display: flex;
+        align-items: center;
+        margin-bottom: ${1 * GU}px;
+      `}
+    >
+      <div
+        css={`
+          background: ${color};
+          width: ${1 * GU}px;
+          height: ${1 * GU}px;
+          border-radius: 50%;
+          margin-right: ${0.5 * GU}px;
+        `}
+      />
+      <div
+        css={`
+          background: ${theme.surfaceUnder};
+          padding: 0px ${1 * GU}px;
+          border-radius: 4px;
+        `}
+      >
+        <img src={tokenIconUrl(label)} alt="" height={16} width={16} />
```

Comment:
`<img src={tokenIconUrl(label)} alt="" height="16" width="16" />`

i think i can not suggest on closed prs